great_expectations/render/renderer_configuration.py

```diff
@@ -104,7 +105,7 @@ def dict(
 
 RendererValueTypes: TypeAlias = Union[RendererValueType, List[RendererValueType]]
 
-AddParamArgs: TypeAlias = Tuple[str, RendererValueTypes]
+AddParamArgs: TypeAlias = Tuple[Tuple[str, RendererValueTypes], ...]
```

Comment:
`non-blocking`
Would a named tuple be helpful for this inner tuple?

Reply:
I don't see why not. ✅

Reply:
That's what I used!